CHANGELOG.latest.md

```diff
@@ -1,3 +1,47 @@
-⚠️⚠️ This is a pre-release version. ⚠️⚠️
+#### Apple AdServices support
```

Comment:
Having this first makes it seem like it's the main change in the release. Maybe we should move this down?

Reply:
+1, let's get Amazon support first, then AdServices, then StoreKit 2 (StoreKit 2 is great but ultimately is under-the-hood stuff that we'll use on behalf of the dev to make stuff more reliable)